.travis.yml

```diff
@@ -3,14 +3,5 @@ sudo: required
 services:
   - docker
 
-# Travis has docker CLI installed but we need a later version
```

Comment:
That config is unrelated to `docker` it's for `docker-compose` as we don't need `docker-compose` to deploy to Rancher, this shouldn't be in this repo. It might be needed for a application tests, which is probably why it crept into here.